### ISSUE TITLE
Suggestion: Differentiate Towns in a Nation from Allies for Perms

### ISSUE THREAD

**Issue by BlazeKnight-x** [TownyAdvanced/Towny#3241]
Right now, there is no way to make a town open to the nation without making it open to the nation's allies. Adding a new perm group called "nation" would be nice for those that want to make their town public to the nation only and not the nation's allies.

**Comment by LlmDl** [comment]
So plot perms would be B=RNAO, D=RNAO, etc

**Comment by BlazeKnight0** [comment]
Essentially, yes. All perms should be togglable with allies in mind

**Comment by LlmDl** [comment]
Done as of https://github.com/TownyAdvanced/Towny/commit/455c0fa1588740db7ba5078a95f5d4126313914a and will be a part of 0.95.0.1.

```
  - New Perm Levels: Town & Nation
    - On player-owned plots the perm line now has Town added to it.
      - Build/Destroy/Switch/ItemUse can be set for friends/townmembers/allies/outsiders.
      - Allies consists of nation-members and nation-allies.
    - On town-owned plots the perm line now has Nation added to it.    
      - Build/Destroy/Switch/ItemUse can be set for residents/nationmembers/allies/outsiders.
      - Allies consists of nation-allies.
    - Closes ticket #3241
```
